README.md

```diff
@@ -109,6 +109,18 @@ To skip subsampling and use all records that you downloaded from GISAID, set `fi
 
 Explore the other configuration files in `profiles/`, to see other examples of how you can build your own Nextstrain workflows for influenza.
 
+> [!IMPORTANT]
+> The workflow is optimized for HA and NA segments and requires additional files if you are building other segments!
+
+- The following files are required for different lineage and segment builds:
+  - reference: "config/{lineage}/{segment}/reference.fasta"
+  - annotation: "config/{lineage}/{segment}/genemap.gff"
+  - tree_exclude_sites: "config/{lineage}/{segment}/exclude-sites.txt"
```

Comment:
Actually, do we even use the `tree_exclude_sites` config? All the [references](https://github.com/search?q=repo%3Anextstrain%2Fseasonal-flu%20tree_exclude_sites&type=code) I can find are config YAMLs, nothing in the workflow uses this param.